docs/guides/model-your-first-process.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: model-your-first-process
```

Comment:
I'm struggling to understand the difference between this doc and https://docs.camunda.io/docs/guides/automating-a-process-using-bpmn/ -- is the latter just a bit more sophisticated? If so, would likely use this as a next step.

Reply:
Modeling (this page) = drawing a first process

Automating ([that page](https://docs.camunda.io/docs/guides/automating-a-process-using-bpmn/)) = Creating an executable process